src/ArrayOfArraysView.hpp

```diff
@@ -33,8 +33,8 @@
  * @note This is only active when LVARRAY_BOUNDS_CHECK is defined.
  */
 #define ARRAYOFARRAYS_CHECK_BOUNDS( i ) \
-  LVARRAY_ERROR_IF( !arrayManipulation::isPositive( i ) || i >= this->size(), \
-                    "Bounds Check Failed: i=" << i << " size()=" << this->size() )
+  LVARRAY_ERROR_IF_PRINTF( !arrayManipulation::isPositive( i ) || i >= this->size(), \
+                           "Bounds Check Failed: i=%" PRId64 " size()=%" PRId64, (int64_t)i, (int64_t)this->size() )
```

Comment:
where is `PRId64` defined, and what if the index is not 64 bit?

Reply:
PRId64 is in inttypes.h https://www.cplusplus.com/reference/cinttypes/,
Int are casted into int64 to be sure not to have overflow.